server/NewBank.java

```diff
@@ -62,12 +75,12 @@ public synchronized String processRequest(CustomerID customer, String request, N
 		return "FAIL";
 	}
 
-	public HashMap<String, Customer> getCustomers() {
+	public HashMap<String, newbank.server.Customer> getCustomers() {
```

Comment:
The `newbank.server` is not necessary, and actually makes the code harder to read, please remove it